apps/desktop/e2e/tests/smoke.spec.ts

```
@@ -0,0 +1,42 @@
+  let fixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await launchApp();
+  });
+
+  test.afterAll(async () => {
+    await closeApp(fixture);
+  });
+
+  // eslint-disable-next-line no-empty-pattern
+  test.afterEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)